main.js

```diff
@@ -0,0 +1,140 @@
+*/
+const form = document.getElementById('contactForm');
+const meal = document.getElementsByName('meal');
+
+/*This if statement checks the value of the selected meal and if
+it is empty removes the 'visible' class from the form. If breakfast
+is selected the visible class will be added to that option only, if
+lunch is selected the visible class will be added to that option only,
+if dinner is selected the visible class will be added to that option.
+*/
+
+if (meal.value == '') {
+    mealOption.classList.remove('hidden');
+}
+    else if (meal.value == 'breakfast'){
+        const breakfastOptions = getElementById('breakfastOptions');
+        breakfastOptions.classList.remove('hidden');
+        breakfastOptions.classList.add('visible');
+    }
+        else if (meal.value == 'lunch'){
+            const lunchOptions = getElementById('lunchOptions');
+            lunchOptions.classList.remove('hidden');
+            lunchOptions.classList.add('visible');
+        }
+            else if (meal.value == 'dinner') {
+                const dinnerOptions = getElementById('dinnerOptions');
+                dinnerOptions.classList.remove('hidden');
+                dinnerOptions.classList.add('visible');
+            }
+/*These statements assign the elements with id values of firstName and 
+lastName to the variables firstName and lastName, the age element with id 
+age to the age variable, the element with the id firstNameHolder to the
+firstNameHolder variable, the element with the id lastNameHolder to the 
+lastNameHolder variable, the element with id ageHolder to the ageHolder variable, 
+the element with the id emailAddress to the emailAddress variable, 
+the element with name of mealOptions to the mealOption variable, and
+the element with id of mealRating to the variable mealRating.
+*/
+const firstName = document.getElementById('firstName');
+const lastName = document.getElementById('lastName');
+const age = document.getElementById('age');
+const firstNameHolder = document.getElementById('firstNameHolder');
+const lastNameHolder = document.getElementById('lastNameHolder');
+const ageHolder = document.getElementById('ageHolder');
+const emailAddress = document.getElementById('emailAdress');
+const mealOption = document.getElementsByName('mealOptionsHolder');
+const mealRating = document.getElementById('mealRating');
+/*This is the event listener to listen for the blur event on the firstNameHolder
+element. It envokes the setNames function.
+*/
+firstNameHolder.addEventListener('blur', function() {
+    setNames();
+});
+/*This is the event listener to listen for the blur event on the lastNameHolder
+element. It envokes the setNames function.
+*/
+lastNameHolder.addEventListener('blur', function() {
+    setNames();
+});
+/*This is the setNames function that takes the input on the element and sets 
+the first character to upper case.
+*/
+function setNames(){
+    let input = document.getElementsByTagName('input');
```

Comment:
you still have the issue where you are looking to all "input" tags. How could you make this more specific? Also - the name of the 2 tags you are wanting to set is not input - but firstname and lastname...maybe you could use an argument? Also - I see you are setting the first char to upper - don't forget to set the rest to lower.